R/AAADefs.R

```diff
@@ -19,11 +19,11 @@
 
 ## common definitions for arules
 .types <- function(method = "apriori") {
-    targets     <- c("frequent itemsets", "maximally frequent itemsets", 
+    targets     <- c("frequent itemsets", "maximally frequent itemsets", "generator frequent itemsets", 
         "closed frequent itemsets", "rules", "hyperedgesets")
     methods     <- c("apriori", "eclat")
     method      <- match.arg(tolower(method), methods)
-    if (method == "eclat") return(targets[1:3])
+    if (method == "eclat") return(targets[1:4])
```

Comment:
/*Makh2018: Change size targets to 4 in order to take generator itemset as target for eclat*/